src/core/selection/selection.ts

```diff
@@ -5,6 +5,4 @@
  * https://github.com/eclipsesource/jsonforms-editor/blob/master/LICENSE
  * ---------------------------------------------------------------------
  */
-import { SchemaElement } from '../model';
-
-export type SelectedElement = SchemaElement | undefined;
+export type SelectedElement = { uuid: string } | undefined;
```

Comment:
is there a reason to wrap the `uuid` in an object?

Reply:
I did this intentionally, because it more clear what the selection is supposed to be and because if we ever need to extend the SelectedElement type we can just add new properties to it without changing any code that expects the selection to contain an `uuid`